### PR TITLE
Feature/fix terraform validate

### PR DESCRIPTION
**Pull request by cathex-ryan-deamer**

### CODE REVIEW

File: pre_commit_python_hooks/remove_provider.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    args = parser.parse_args(argv)
+
+    for arg in args.filenames:
+        folder = re.search("(.*/).*.tf", arg)
+
+        provider_location = folder.group(1) + "provider.tf"
+        if not os.path.isfile(provider_location):
+            ISSUE = "Failed"
+
+            print(f"{ISSUE} theres no proider.tf")
```
Comment (cathex-sam-holdaway) [inline pre_commit_python_hooks/remove_provider.py]: Minor typo there... Missing the v in provider

**Review by cathex-sam-holdaway — CHANGES_REQUESTED** [review]

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -62,3 +62,24 @@
   entry: tf-module-checker
   language: python
   files: .tf$
+
+- id: tf-provider-gen
+  name: Generate tf provider 
+  description: Generate a provider file for terrform validate step.
```
Comment (cathex-matt) [inline .pre-commit-hooks.yaml]: typo on `terrform`

File: pre_commit_python_hooks/remove_provider.py (hunk 1 above)
Comment (cathex-ryan-deamer) [reply pre_commit_python_hooks/remove_provider.py]: Fixed :)

File: .pre-commit-hooks.yaml (hunk 2 above)
Comment (cathex-ryan-deamer) [reply .pre-commit-hooks.yaml]: Fixed :)

**Review by cathex-sam-holdaway — APPROVED** [review]

**Pull request merged**